src/Info.ts

```diff
@@ -0,0 +1,107 @@
+import { fileBackedObject } from "./FileBackedObject";
+import { SharedSettings } from "./SharedSettings";
+
+import Discord = require("discord.js");
+
+// If the Collection contains anything in the other array
+const findOne = (arr1: Discord.Collection<string, Discord.Role>, arr2: Array<any>) => {
+    return arr1.some(v => {
+        return arr2.indexOf(v) >= 0;
+    });
+};
+
+export interface InfoData {
+    Command: string;
+    Message: string;
+}
+
+export default class Info {
+    private bot: Discord.Client;
+    private infos: InfoData[];
+    private sharedSettings: SharedSettings;
+
+    constructor(bot: Discord.Client, sharedSettings: SharedSettings, userFile: string) {
+        console.log("Requested Thinking extension..");
```

Comment:
This should be changed to the extension name

Reply:
Oh no, you noticed my copy/paste work :(